src/main/java/com/sfuentes/Employee.java

```diff
@@ -0,0 +1,37 @@
+package com.sfuentes;
+
+public class Employee {
+
+  int employeeId;
+  final double baseSalary = 10000.00;
+  final double bonusPercentage = 10;
+  double[] listSales;
+  double netSalary;
+
+  public Employee(int employeeId, double[] listSales) {
+    this.employeeId = employeeId;
+
+    if (listSales.length == 3)
+      this.listSales = listSales;
+    else
+      throw new IllegalArgumentException("Incorrect amount of sales");
+  }
+
+  public double calculateSumSalary() {
```

Comment:
the name of this method could be more accurate, for example calculateSalesBonus() since this method is not calculating salary.

Reply:
fixed